### ISSUE TITLE
thoughts on editing timezones in TCX files

### ISSUE THREAD

**Issue by nickpdawson** [cpfair/tapiriik#283]
My workflow currently includes: 
Wahoo fitness -> TCX to dropbox Wahoo folder -> Zapier copies file to Tapirikk folder ->Tapiriik tries to ingest it into Garmin 

But it fails due to timezone issues. 

(I am able to import the file into Garmin Connect using their +Import feature) 

I'm not a proficient coder so I'm not able to suggest a fix or change or PR. But I think I can certainly insert a timezone field into the XML file. However, I cannot find any info on how time zone is defined in the TCX format. 

Does anyone have any ideas on what that might look like?

**Comment by funwhilelost** [comment]
The TCX file format, by default, uses the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) format.  This is usually in GMT.  Can you attach a copy of the TCX from Wahoo?  It's probably a little export problem.

**Comment by nickpdawson** [comment]
Happy to! 
TCX attached 
I though I could get clever and do a little sed manipulation of the file...but it turns out I'm not clever enough ;) 

[2017-01-10-0558_Indoor_Bike_Trainer_WF.tcx.zip](https://github.com/cpfair/tapiriik/files/697449/2017-01-10-0558_Indoor_Bike_Trainer_WF.tcx.zip)

**Comment by funwhilelost** [comment]
Yeah, that data looks good.  The timestamps are all similar to `2017-01-10T10:10:23Z` which looks fine to me.  I wonder if the "timezone issue" is a red herring.  I'm not familiar with the Tapiriik code, just hoping I could help.
